submissions/krylenger/html-css-popup/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pop-up</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+    <nav class="navigation-container">
+        <div class="navigation-container__body">
```

Comment:
I think this container is redundant here because you have only the `ul` element inside your `nav` tag

Reply:
✅ done